n3fit/src/n3fit/backends/keras_backend/MetaModel.py

```diff
@@ -111,13 +115,12 @@ def __init__(self, input_tensors, output_tensors, scaler=None, input_values=None
         super().__init__(input_tensors, output_tensors, **kwargs)
 
         self.x_in = x_in
-        self.tensors_in = input_tensors
+        self.input_tensors = input_tensors
```

Comment:
Please add here as well something like this:

```suggestion
        self.input_tensors = input_tensors
        self.single_replica_generator = None
```